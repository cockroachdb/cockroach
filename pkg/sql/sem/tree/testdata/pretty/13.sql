SELECT ((1)), 2
