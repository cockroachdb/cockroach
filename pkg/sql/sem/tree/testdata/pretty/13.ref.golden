// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	1,
	2

8:
--------
SELECT
	1, 2

11:
-----------
SELECT 1, 2


